src/sys/select.rs

```diff
@@ -110,7 +106,7 @@ impl<'fd> FdSet<'fd> {
     pub fn fds(&self, highest: Option<RawFd>) -> Fds {
         Fds {
             set: self,
-            range: 0..highest.map(|h| h as usize + 1).unwrap_or(FD_SETSIZE),
+            range: 0..highest.map(|h| h + 1).unwrap_or(FD_SETSIZE) as usize,
```

Comment:
Interesting, they changed the type of this constant from `usize` to `c_int`: https://github.com/rust-lang/libc/pull/3356/files